hack/test-utils.sh

```diff
@@ -40,13 +40,14 @@ test_setup() {
   fi
 
   # Start containerd
-  if [ ! -x "$(command -v containerd)" ]; then
+  containerd=$(command -v containerd)
+  if [ ! -x "${containerd}" ]; then
     echo "containerd is not installed, please run hack/install-deps.sh"
     exit 1
   fi
   sudo pkill -x cri-containerd
   sudo pkill -x containerd
-  keepalive "sudo containerd" ${RESTART_WAIT_PERIOD} &> ${report_dir}/containerd.log &
+  keepalive "sudo ${containerd}" ${RESTART_WAIT_PERIOD} &> ${report_dir}/containerd.log &
```

Comment:
Agree we should test for and run the same containerd, but a bit concerned someone may have two versions of containerd one on his sudo path and another on his current user path and thus this change might introduce the situation where we are not running the one he thinks we are running. For example, there might be an older version of containerd over in "home/username/go/bin"

Reply:
I think this change makes sense.

If you are worried about the 2 containerd case, can we `echo` something after `command -v containerd` to tell developer what containerd is being used? e.g.
```
using /usr/local/bin/containerd
```

Reply:
echoing the path is nice as is echoing the version used.. 

But would still rather not use a mix of of execution paths.. if we are going to run with the user path use `PATH=$PATH` otherwise IMO it's better to just fix the test condition.

Reply:
That also works for me, echo + `PATH=$PATH`.

Reply:
@jcvenegas Please echo the path of containerd we are using in the test.
And then I think the PR is good to go. :)

Reply:
@Random-Liu PR updated, @mikebrow added a comment as alternative to do not mix execution paths. Let me know if you prefer the alternative proposal.